e2e/testcases/cli_test.go

```diff
@@ -833,3 +848,235 @@ func TestNomosVetNamespaceRepo(t *testing.T) {
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/namespaces/config-management-system/root-reconciler.*/hydration-controller.txt", bugReportDirName, context), files)
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/namespaces/config-management-system/root-reconciler.*/otel-agent.txt", bugReportDirName, context), files)
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/namespaces/config-management-system/root-reconciler.*/reconciler.txt", bugReportDirName, context), files)
+
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/namespaces/config-management-monitoring/pods.txt", bugReportDirName, context), files)
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/namespaces/config-management-monitoring/pods_yaml.txt", bugReportDirName, context), files)
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/namespaces/config-management-monitoring/otel-collector.*/otel-collector.txt", bugReportDirName, context), files)
+
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/namespaces/gatekeeper-system/pods.txt", bugReportDirName, context), files)
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/namespaces/gatekeeper-system/pods_yaml.txt", bugReportDirName, context), files)
+
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/namespaces/resource-group-system/pods.txt", bugReportDirName, context), files)
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/namespaces/resource-group-system/pods_yaml.txt", bugReportDirName, context), files)
+
+	} else {
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/namespaces/config-management-system/git-importer.*/git-sync.txt", bugReportDirName, context), files)
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/namespaces/config-management-system/git-importer.*/git-sync.txt", bugReportDirName, context), files)
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/namespaces/config-management-system/git-importer.*/importer.txt", bugReportDirName, context), files)
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/namespaces/config-management-system/monitor.*/monitor.txt", bugReportDirName, context), files)
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/namespaces/config-management-system/pods.txt", bugReportDirName, context), files)
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/namespaces/kube-system/pods.txt", bugReportDirName, context), files)
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/cluster/configmanagement/clusterconfigs.txt", bugReportDirName, context), files)
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/cluster/configmanagement/namespaceconfigs.txt", bugReportDirName, context), files)
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/cluster/configmanagement/repos.txt", bugReportDirName, context), files)
+		found = found && checkFileExists(fmt.Sprintf("%s/raw/%s/cluster/configmanagement/syncs.txt", bugReportDirName, context), files)
```

Comment:
Do sth to avoid duplication.

Reply:
We want to validate all these files exist under the specific path, and the folder structure is different when running under mono or multi repo mode.

Reply:
I see.

Reply:
Can we define a list of expected files and then iterate through the list using a for loop?

```
    expectedFiles := []string{
		fmt.Sprintf("%s/processed/%s/version.txt", bugReportDirName, context),
		fmt.Sprintf("%s/processed/%s/status.txt", bugReportDirName, context),
	}
	if nt.MultiRepo {
		expectedFiles = append(expectedFiles, ...)
		...
	} else {
		expectedFiles = append(expectedFiles, ...)
		...
	}

	for _, file := range expectedFiles {
		if checkFileExists(file, files) {
			nt.T.Errorf("did not find all expected files in bug report zip file")
		}
	}

```

Using the above code should get rid of the `found` flag.

Reply:
Done!